duckduckgo/Android
# Feature/hide privacy grade until data loaded

Asana Issue URL: https://app.asana.com/0/414730916066338/514214694717656

## Description
Hides the privacy grade button until the app configuration data (easylist tracker etc..) has been downloaded.

This will mean that the user is unprotected while this is happening. This isn't a change in functionality, but worth remembering, as we might want to alert the user in future.

Before the data is downloaded, no privacy grade button will be visible. After the data downloads and is initialised, the privacy grade button will be shown **on the next URL change**.

## Steps to Test this PR:
1. Fresh install
1. Make a search
1. Verify the privacy grade button is missing
1. Wait for app config to download (look in logcat for `Download task completed successfully`)
1. Change url by navigating to a different page
1. Verify privacy grade button is visible
